github_jira_sync_app/instrumentation/metrics.py

```diff
@@ -0,0 +1,113 @@
+import logging
+import os
+import time
+
+from fastapi import FastAPI
+from fastapi import Request
+from fastapi.responses import JSONResponse
+from opentelemetry import metrics
+from opentelemetry.exporter.prometheus import PrometheusMetricReader
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.resources import SERVICE_NAME
+from opentelemetry.sdk.resources import Resource
+from prometheus_client import make_asgi_app
+
+
+def define_logger():
```

Comment:
should we have one logger for the whole service?
otherwise we have code duplication